pymoab/types.pyx

```diff
@@ -54,6 +56,52 @@ def check_error(int err, tuple exceptions, **kwargs):
         msg += ', '.join(sorted(['{0}={1!r}'.format(k, v) for k, v in kwargs.items()]))
     raise errtype(msg)
 
+# Data Types
+MB_TYPE_OPAQUE  = moab.MB_TYPE_OPAQUE
+MB_TYPE_INTEGER  = moab.MB_TYPE_INTEGER
+MB_TYPE_DOUBLE   = moab.MB_TYPE_DOUBLE
+MB_TYPE_BIT      = moab.MB_TYPE_BIT
+MB_TYPE_HANDLE   = moab.MB_TYPE_HANDLE
+MB_MAX_DATA_TYPE = moab.MB_MAX_DATA_TYPE
+
+_DTYPE_CONV = {
+    MB_TYPE_OPAQUE   : 'S',
+    MB_TYPE_INTEGER  : 'int32',
+    MB_TYPE_DOUBLE   : 'float64',
+    MB_TYPE_BIT      : 'bool',
+    MB_TYPE_HANDLE   : 'uint64',
+    MB_MAX_DATA_TYPE : 'uint64'
+    }
+
+
+_VALID_DTYPES= {
```

Comment:
I think that you can do `cdef dict _VALID_DTYPES = {` for these dicts if you don't care about them being accessible from Python.